bolt/test/X86/Inputs/dwarf5-df-input-lowpc-ranges-other.s

```diff
@@ -0,0 +1,453 @@
+# clang++ -fbasic-block-sections=all -ffunction-sections -g2 -gdwarf-5 -gsplit-dwarf
```

Comment:
Was it generated with: -fdebug-compilation-dir

Reply:
Yes, forgot to change the comment at the top. Will fix.